modules/virt-cluster-resource-requirements.adoc

```diff
@@ -49,16 +53,21 @@ Calculate the cluster processor overhead requirements for {VirtProductName} by u
 .Cluster CPU overhead
 
 ----
-CPU overhead for infrastructure nodes ≈ 4
+CPU overhead for infrastructure nodes ≈ 4 cores
 ----
 
+{VirtProductName} increases the overall utilization of cluster level services such as logging, routing, and monitoring. To account for this workload, ensure that nodes hosting infrastructure components have capacity allocated for 4 additional cores (4000 millicores) across all nodes.
+
 ----
-CPU overhead for worker nodes ≈ 2 + CPU overhead per virtual machine
+CPU overhead for worker nodes ≈ 2 cores + CPU overhead per virtual machine
 ----
 
+Each worker node that hosts virtual machines must have capacity for 2 additional cores (2000 millicores) for {VirtProductName} management workloads in addition to the CPUs required for virtual machine workloads.
+
+
 .Virtual machine CPU overhead
 
-CPU overhead per virtual machine is subjective. If dedicated CPUs are requested, there is a 1:1 impact on the cluster CPU overhead requirement. Otherwise, there are no specific rules about how many CPUs a virtual machine requires.
+If dedicated CPUs are requested, there is a 1:1 impact on the cluster CPU overhead requirement. When using virtual CPUs, the default CPU allocation ratio is 10 vCPUs:1 physical CPU.
```

Comment:
I think I recognize that the 10:1 ratio comes from the VMs only requesting 100 millicpu for each `virt-launcher` Pod, but I'm afraid that including this information will be confusing and lead to more questions. What if I only want 4:1 overcommitment? How do I enforce overcommitment rules? Isn't overcommitment bad? And so on. 

I think it may be simplest to simply state something along the lines of the default CPU request is 100 millicpu for each running virtual machine, with real CPU utilization being no more than the VM's CPU allotment.

Oversubscription is a big topic, particularly when resource contention occurs and results in Pod evictions and/or the descheduler taking action. Perhaps in the future we should add a section to give additional data and empower the reader with enough context to make informed decisions.

Reply:
Agreed that there is a separate discussion for CPUAllocationRatio where all the nuance you just mentioned comes into play. Perhaps it's best for the purposes of the "what's the minimum I need" discussion, go back to our original wording where we don't offer specific recommendations for non-dedicated vCPUs.